### PR TITLE
Corrected Atrophy Set & Voltsurge Torque

### PR DESCRIPTION
**Pull request by R3P0FFXI**
Corrected multiple errors in Atrophy Set, added mods for  Voltsurge Torque

### CODE REVIEW

**Comment by takhlaq** [comment]
> Importing item_mods.sql into the database...ERROR 1064 (42000) at line 34693: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '--mp
INSERT INTO `item_mods` VALUES (28094,8,16)' at line 1

https://travis-ci.org/DarkstarProject/darkstar/builds/167939219#L2411

**Comment by R3P0FFXI** [comment]
@takhlaq I saw it and my git is now screwed up I am waiting for @TeoTwawki to see if we can fix my local copy so I can amend the commit or if I need to redo the entire thing.

**Comment by takhlaq** [comment]
hard revert to `949d9df`
then fix, commit > amend last commit > force push

File: sql/item_mods.sql [hunk 1]
```diff
@@ -36104,22 +36108,22 @@ INSERT INTO `item_mods` VALUES (28247,29,5);
...
+INSERT INTO `item_mods` VALUES (28248,2,48); -- hp
+INSERT INTO `item_mods` VALUES (28248,5,49); -- mp
+INSERT INTO `item_mods` VALUES (28248,8,10); -- str
+INSERT INTO `item_mods` VALUES (28248,9,11); -- dex
+INSERT INTO `item_mods` VALUES (28248,10,10); -- vit
+INSERT INTO `item_mods` VALUES (28248,11,33); -- agi
+INSERT INTO `item_mods` VALUES (28248,12,17); -- int
+INSERT INTO `item_mods` VALUES (28248,13,19); -- mnd
+INSERT INTO `item_mods` VALUES (28248,14,34); -- chr
+INSERT INTO `item_mods` VALUES (28248,25,20) -- acc
```
Comment (R3P0FFXI) [inline sql/item_mods.sql]: Ugh damn I suck, fixed and otw

**Pull request merged**